#the nav file name
string file_name

#specify the floor name,a nav map may contain multy floors
string floor_name

#the waypoint index of lane
uint16 start_index
uint16 end_index

#if the lane is double-side
bool double_side
---
#the bool flag to judge if set success
bool success
uint8 index